care/facility/api/serializers/patient.py

```diff
@@ -104,6 +111,11 @@ def validate_facility(self, value):
             raise serializers.ValidationError("facility not found")
         return value
 
+    def validate_countries_travelled(self, value):
+        if value and not type(value) is list:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using type() instead of isinstance() for a typecheck.](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5380979)